fields.yml

```diff
@@ -0,0 +1,1215 @@
+---
+- key: ecs
+  title: ECS
+  description: >
+    ECS fields.
+  fields:
+  
+  - name: agent
```

Comment:
Would it be easy to move this over another 2 spaces? My goal is to have the it look like this

```
fields:
  - blah
```

rather than

```
fields:
- blah
```

which is consistent with indentations found in other areas of the file.